create_appimage.sh

```diff
@@ -3,8 +3,8 @@ if [[ "$BUILDARCH" == "x64" ]]; then
   # install a dep needed for this process
   sudo apt-get install desktop-file-utils
 
-  # download pkg2appimage from github
-  curl -LO "https://github.com/AppImage/pkg2appimage/raw/master/pkg2appimage"
+  # download modified pkg2appimage from gist
+  curl -LO "https://gist.githubusercontent.com/tyu1996/39b8a7a29c2cdeef882daaa1702ec971/raw/2e788010deafb2f1214439ee0bc8e46685480e45/pkg2appimage"
```

Comment:
Is there any way we can avoid doing this? What about if we put the modified pkg2appimage directly in this repo?

Reply:
Is it something regarding the gist?

pkg2appimage requires functions.sh which requires AppRun to complete the AppImage packing, I modified all three files just for that. Is it encouraged if I put all these 3 files in this repo?

Reply:
Yes; I would be ok keeping our modified pkg2appimage files in a folder in this repo.